resources/zib-AddressInformation.xml

```diff
@@ -0,0 +1,339 @@
+         <path value="Address.district"/>
+         <short value="Municipality"/>
+         <definition value="Municipality of residence. For Dutch municipalities, preferably use the name from the GBA, table 33 (OID: 2.16.840.1.113883.2.4.6.14)."/>
+         <comment value="For Dutch municipalities, preferably use the name from the GBA, table 33 (OID: 2.16.840.1.113883.2.4.6.14)."/>
+         <alias value="Gemeente"/>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.4"/>
+            <comment value="Municipality"/>
+         </mapping>
+      </element>
+      <element id="Address.postalCode">
+         <path value="Address.postalCode"/>
+         <short value="Postcode"/>
+         <definition value="Postcode of the address. In Dutch addresses, preferably use the postcode from the Postcode table (OID: 2.16.840.1.113883.2.4.4.15)."/>
+         <comment value="In Dutch addresses, preferably use the postcode from the Postcode table (OID: 2.16.840.1.113883.2.4.4.15)."/>
+         <alias value="Postcode"/>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.6"/>
+            <comment value="Postcode"/>
+         </mapping>
+      </element>
+      <element id="Address.country">
+         <path value="Address.country"/>
+         <comment value="The FHIR datatype does not properly support coding the country using a CodeableConcept or Coding datatype. If the sender supports coded countries, it SHALL send the coded value using the code-specification extension under Address.country."/>
+      </element>
+      <element id="Address.country.extension">
+         <path value="Address.country.extension"/>
+         <slicing>
+            <discriminator>
+               <type value="value"/>
+               <path value="url"/>
+            </discriminator>
+            <rules value="open"/>
+         </slicing>
+      </element>
+      <element id="Address.country.extension:countryCode">
+         <path value="Address.country.extension"/>
+         <sliceName value="countryCode"/>
+         <type>
+            <code value="Extension"/>
+            <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-code-specification"/>
+         </type>
+      </element>
+      <element id="Address.country.extension:countryCode.value[x]">
+         <path value="Address.country.extension.value[x]"/>
+         <short value="Country"/>
+         <definition value="Country in which the address is located."/>
+         <alias value="Land"/>
+         <mapping>
+            <identity value="zib-addressinformation-v1.1-2020EN"/>
+            <map value="NL-CM:20.5.5"/>
+            <comment value="Country"/>
+         </mapping>
+      </element>
+      <element id="Address.country.extension:countryCode.value[x].coding">
+         <path value="Address.country.extension.value[x].coding"/>
+         <slicing>
```

Comment:
Hier zouden we voor gecombineerde ValueSets ipv slicing gaan toch? Of is er een goede reden om hiervan af te wijken?

Reply:
Klopt, gefixt